lib/src/dependency.dart

```diff
@@ -35,7 +36,7 @@ Map<String, Dependency> parseDeps(Map source) =>
 const _sourceKeys = ['sdk', 'git', 'path', 'hosted'];
 
 /// Returns `null` if the data could not be parsed.
-Dependency _fromJson(dynamic data) {
+Dependency? _fromJson(dynamic data) {
```

Comment:
Force any dynamic calls to be explicit casts with:

```suggestion
Dependency? _fromJson(Object? data) {
```

Reply:
gotcha